### PR TITLE
Rds/cml/19 5

### PR DESCRIPTION
**Pull request by claperle**

### CODE REVIEW

File: docs/section19/19-5.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+1. Applies only to the proposed design rule set model instance.  
+
+**Function Calls:**  None
+
+## Rule Logic:   
+**Applicability Check 1:**  
+- For each RulesetModelInstance in ASHRAE229.ruleset_model_instance : `for RulesetModelInstance in ASHRAE229.ruleset_model_instance:`  
+    - Check if the ruleset model instance is the proposed model instance: `If RulesetModelInstance.RulesetModelOptions2019ASHRAE901.ruleset_model_type = "PROPOSED":`  
+        - Get the heating unmet load hours for the rule set model instance: `unmet_load_hours_heating = RulesetModelInstance.OutputInstance.unmet_load_hours_heating`  
+        - Get the cooling unmet load hours for the rule set model instance: `unmet_load_hours_cooling = RulesetModelInstance.OutputInstance.unmet_load_hours_cooling`
```
Comment (weilixu) [inline docs/section19/19-5.md:25]: There is no output instance yet - we will need to wait for the latest schema that indicates the correct jsonpath to retrieve the `unmet_load_hours_` results from output format. This needs to be confirmed with @JasonGlazer

File: docs/section19/19-5.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Section 19 - Rule 19-5      
+**Schema Version:** 0.0.22
```
Comment (weilixu) [inline docs/section19/19-5.md:2]: Should update this to 0.0.23 to include the latest change in the schema

File: docs/section19/19-5.md (hunk 1 above)
Comment (weilixu) [reply docs/section19/19-5.md:25]: This is the output path to unmet_load_hours...

`RulesetModelInstance.output.OutputInstance.unmet_load_hours_heating`
`RulesetModelInstance.output.OutputInstance.unmet_load_hours_cooling`

File: docs/section19/19-5.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    - Check if the ruleset model instance is the proposed model instance: `If RulesetModelInstance.RulesetModelOptions2019ASHRAE901.ruleset_model_type = "PROPOSED":`  
+        - Get the heating unmet load hours for the rule set model instance: `unmet_load_hours_heating = RulesetModelInstance.OutputInstance.unmet_load_hours_heating`  
+        - Get the cooling unmet load hours for the rule set model instance: `unmet_load_hours_cooling = RulesetModelInstance.OutputInstance.unmet_load_hours_cooling`  
+
+        **Rule Assertion:** 
+        - Case 1: If the heating + cooling unmet load hours is 300 or less then pass: `if unmet_load_hours_heating + unmet_load_hours_cooling <= 300: outcome = "PASS"`   
+        - Case 2: Else, fail: `Else: outcome = "Fail"`  
+
+**Notes/Questions:**  
+1. Does this apply at the ruleset model instance level as I created it? I wasn't sure how to reference the output schema. I took a shot at it here.
```
Comment (weilixu) [inline docs/section19/19-5.md:32]: I agree this shall be at ruleset model instance level.

**Review by weilixu — APPROVED** [review]
Approved after updating the output path

File: docs/section19/19-5.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+## Rule Logic:   
+**Applicability Check 1:**  
+- For each RulesetModelInstance in ASHRAE229.ruleset_model_instance : `for RulesetModelInstance in ASHRAE229.ruleset_model_instance:`  
+    - Check if the ruleset model instance is the proposed model instance: `If RulesetModelInstance.RulesetModelOptions2019ASHRAE901.ruleset_model_type = "PROPOSED":`  
+        - Get the heating unmet load hours for the rule set model instance: `unmet_load_hours_heating = RulesetModelInstance.output.OutputInstance.unmet_load_hours_heating`  
+        - Get the cooling unmet load hours for the rule set model instance: `unmet_load_hours_cooling = RulesetModelInstance.output.OutputInstance.unmet_load_hours_cooling`  
+
+        **Rule Assertion:** 
+        - Case 1: If the heating + cooling unmet load hours is 300 or less then pass: `if unmet_load_hours_heating + unmet_load_hours_cooling <= 300: outcome = "PASS"`   
+        - Case 2: Else, fail: `Else: outcome = "Fail"`
```
Comment (mkarpman) [inline docs/section19/19-5.md:29]: If during a given hour there are both unmet heating and unmet cooling loads, the hour will be conted under both  unmet_load_hours_heating AND unmet_load_hours_cooling, so will be double-counted. To address this, I suggest modifying Case to FAIL if >600 and adding Cased 3 for 301-600 unmet hours with UNDETERMINED outcome and a clarifying message. Another option is to add coincident_unmet_load_hours to the schema and base the RDS on that value.

File: docs/section19/19-5.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+        **Rule Assertion:** 
+        - Case 1: If the coincident unmet load hours are less than or equal to 300 and not equal to Null then pass: `if coincident_unmet_load_hours != Null and coincident_unmet_load_hours <= 300: outcome = "Pass"`  
+        - Case 2: Elif the heating + cooling unmet load hours less than or equal to 300 and neither is equal to Null then pass: `elif unmet_load_hours_heating != Null and unmet_load_hours_cooling != Null and (unmet_load_hours_heating + unmet_load_hours_cooling <= 300): outcome = "Pass"`  
+        - Case 3: Elif the coincident unmet load hours are greater than 300 and not equal to Null then fail: `elif coincident_unmet_load_hours != Null and coincident_unmet_load_hours > 300: outcome = "Fail"`  
+        - Case 4: Elif the heating + cooling unmet load hours is less than or equal to 300 and neither is equal to Null then pass: `elif unmet_load_hours_heating != Null and unmet_load_hours_cooling != Null and (unmet_load_hours_heating + unmet_load_hours_cooling <= 300): outcome = "Pass"`
```
Comment (supriyagoel) [inline docs/section19/19-5.md:32]: what is the difference between case 2 and case 4?

File: docs/section19/19-5.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+1. Applies only to the proposed design rule set model instance.  
+
+**Function Calls:**  None
+
+## Rule Logic:   
+**Applicability Check 1:**  
+- For each RulesetModelInstance in ASHRAE229.ruleset_model_instance : `for RulesetModelInstance in ASHRAE229.ruleset_model_instance:`  
+    - Check if the ruleset model instance is the proposed model instance: `If RulesetModelInstance.RulesetModelOptions2019ASHRAE901.ruleset_model_type = "PROPOSED":`  
+        - Get the heating unmet load hours for the rule set model instance: `unmet_load_hours_heating = RulesetModelInstance.output.OutputInstance.unmet_load_hours_heating`  
+        - Get the cooling unmet load hours for the rule set model instance: `unmet_load_hours_cooling = RulesetModelInstance.output.OutputInstance.unmet_load_hours_cooling`
```
Comment (supriyagoel) [inline docs/section19/19-5.md:25]: @claperle should we be using the unmet hrs for H and C when occupied?
Similarly, do we need to clarify in the schema notes that 'unmet_load_hours' should be only for the occupied hrs?

**Review by supriyagoel — COMMENTED** [review]
comment on whether H and C unmet hrs when occupied should be used

File: docs/section19/19-5.md (hunk 6 above)
Comment (claperle) [reply docs/section19/19-5.md:25]: Let's discuss on the call. App G does not separate occupied from unoccupied unmet load hours (as far as I know) so we did not include those data elements. I was not sure why they were included in the schema.

**Review by mkarpman — APPROVED** [review]

File: docs/section19/19-5.md (hunk 6 above)
Comment (claperle) [reply docs/section19/19-5.md:25]: Per call on 1/12/2023 - as the RDS is written is good. No changes needed.

**Review by supriyagoel — APPROVED** [review]

**Pull request merged**